core/src/main/java/com/radixdlt/p2p/addressbook/AddressBook.java

```diff
@@ -209,10 +214,10 @@ private void insertOrUpdateAddressBookEntryWithUri(RadixNodeUri uri) {
     final var newOrUpdatedEntry =
         maybeExistingEntry == null
             ? AddressBookEntry.create(uri)
-            : maybeExistingEntry.cleanupExpiredBlacklsitedUris().addUriIfNotExists(uri);
+            : maybeExistingEntry.cleanupExpiredBlacklistedUris().addUriIfNotExists(uri);
```

Comment:
I wonder if this is a good time to `sed/blacklist/blocklist` and `sed/whitelist/allowlist` in the codebase?

Reply:
I think here we could actually rename it to `isInvalid` / `markAsInvalid`.

Reply:
or remove it completely :thinking: we're only blocking the address if handshake fails, so why not remove it immediately?

Reply:
or if we're keeping it, we could name it even more explicitly: `handshakeFailed` or similar

Reply:
I guess it makes _some_ sense to keep an address with this flag in the address book for some time (so that we don't retry if the address is received again in peer discover)